srcpkgs/libwacom/template

```diff
@@ -26,7 +26,8 @@ post_install() {
 }
 
 libwacom-devel_package() {
-	depends="libgudev-devel ${sourcepkg}>=${version}_${revision}"
+	depends="libgudev-devel ${sourcepkg}>=${version}_${revision}
+	 libevdev-devel"
```

Comment:
```suggestion
	depends="${sourcepkg}>=${version}_${revision}"
```

libwacom only `Requires.private: gudev-1.0, glib-2.0, libevdev`  hence, it shouldn't depend on `libgudev-devel` either.